chia/consensus/multiprocess_validation.py

```diff
@@ -314,8 +317,8 @@ async def pre_validate_blocks_multiprocessing(
             # We ONLY add blocks which are in the past, based on header hashes (which are validated later) to the
             # prev blocks dict. This is important since these blocks are assumed to be valid and are used as previous
             # generator references
-            prev_blocks_dict: Dict[uint32, Union[FullBlock, HeaderBlock]] = {}
```

Comment:
it looks like someone typoed `uint32` instead of `bytes32`. I don't know why mypy accepted this.

Reply:
Yep, and `FullBlock.header_hash` and `HeaderBlock.header_hash` are both `@property` that have no return hints thus leaving them as `Any` and allowing this error to go uncaught by mypy.  Exactly why stricter settings are needed everywhere to avoid unhinted stuff and then later to entirely ban the explicit use of `Any` as well.